src/extension.ts

```diff
@@ -69,39 +81,51 @@ const init = async (
     restartLogs(env);
   });
 
-  vscode.commands.registerCommand("env0.openInEnv0", (env) => {
-    openEnvironmentInBrowser(env);
-  });
+  context.subscriptions.push(
+    vscode.commands.registerCommand("env0.openInEnv0", (env) => {
+      openEnvironmentInBrowser(env);
+    })
+  );
```

Comment:
@meniRoy 
can you make some arrangements here and create component/file that exports a map of 
```
Record<string, () => void> commands =  {"env0.openInEnv0", (env) => openEnvironmentInBrowser(env)}
```
and then create a function `loadSubscriptions` :

```
commands.forEach(... // register command and subscription )
```

Reply:
I will open separate PR for that after i will marge the deployment logs PR since they touch the same methods, and I want to avoid unnecessary conflicts